clang/test/CodeGenCXX/debug-sin.cpp

```diff
@@ -1,23 +1,28 @@
 ////////////////////////////////////////////////////////////////////////////////////////
 
-// With default options, ensure that declarations obtained by a using declaration have
+// With -fsystem-debug option, ensure that declarations obtained by a using declaration have
 // debug info generated.  sin() is declared through cmath with: using ::sin;
 //
-// Also ensure that no debug info for sin() is generated if -fno-system-debug is used.
+// Also ensure that no debug info for sin() is generated by default or if -fno-system-debug is used.
 
 // Debug info for math library functions is not generated on Windows
 // UNSUPPORTED: system-windows
 
+// RUN: %clang -fsystem-debug    -emit-llvm -S -g %s     -o %t.system_debug.ll
 // RUN: %clang                   -emit-llvm -S -g %s     -o %t.default.ll
 // RUN: %clang -fno-system-debug -emit-llvm -S -g %s     -o %t.no_system_debug.ll
```

Comment:
I am not sure if we need testing with -fno-system-debug. Ideally, it will be great if we can remove fno-system-debug everything.

Thanks